From d28ae3d4d943ec78626444ac96a7b59b53d94556 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 6 Feb 2025 12:58:31 -0500
Subject: [PATCH] gpg: --gpgconf-list: report actual compliance mode

* g10/gpg.c (gpgconf_list): report actual compliance mode

--

GnuPG-bug-id: 7518
Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 g10/gpg.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/g10/gpg.c b/g10/gpg.c
index dce171efb..32cef0a8d 100644
--- a/g10/gpg.c
+++ b/g10/gpg.c
@@ -1995,7 +1995,8 @@ static void
 gpgconf_list (void)
 {
   es_printf ("debug-level:%lu:\"none:\n", GC_OPT_FLAG_DEFAULT);
-  es_printf ("compliance:%lu:\"%s:\n", GC_OPT_FLAG_DEFAULT, "gnupg");
+  es_printf ("compliance:%lu:\"%s:\n", GC_OPT_FLAG_DEFAULT,
+             gnupg_compliance_label (opt.compliance));
 
   /* The next one is an info only item and should match the macros at
      the top of keygen.c  */
